src/Sylius/Bundle/ShopBundle/Resources/views/Cart/_widget.html.twig

```diff
@@ -5,9 +5,9 @@
 
     <i class="cart icon"></i>
     <span id="sylius-cart-total">
-        {{ money.convertAndFormat(cart.itemsTotal) }}
+        {{- money.convertAndFormat(cart.itemsTotal) -}}
     </span>
-    {% transchoice cart.items|length %}sylius.ui.item.choice{% endtranschoice %}
+    {{- 'sylius.ui.item.choice'|trans({'%count%': cart.items|length}) }}
```

Comment:
Why did you decide to trim this message only from one side?

Reply:
It does the job of removing unnecessary space between price and the number of items - probably should be backported to Sylius 1.5 as a bugfix though.

It looked like "$10.00 , 1 item", now it's "$10.00, 1 item".